src/components/Footer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Link } from "../../styles";
+import {
+  InButton,
+  Container,
+  InIcon,
+  FindUsText,
+  Text,
+  NormalText,
+  GoUpButton,
+  GoUpButtonContainer,
+  Background,
+} from "./style";
+
+const Footer = () => {
+  const handleInBeataClick = () => {
+    window.open("https://www.linkedin.com/in/beatasuscicka/", "_blank");
+  };
+  const handleInNataliaClick = () => {
+    window.open("https://www.linkedin.com/in/natalia-mateusiak/", "_blank");
+  };
+
+  return (
+    <Background>
+      <Container>
+        <GoUpButtonContainer>
+          <Link smooth to="/#header_section">
+            <GoUpButton>go up!</GoUpButton>
+          </Link>
+        </GoUpButtonContainer>
+        <FindUsText>Find us</FindUsText>
+        <InButton onClick={handleInBeataClick}>
```

Comment:
This button is only opening a page in new tab so maybe you could use `a` tag with `href` instead of button